Tests/SonarQube.MSBuild.Tasks.UnitTests/AttachBuildWrapperTests.cs

```diff
@@ -39,7 +45,7 @@ public void AttachBW_MissingFiles_Fails()
                 }
                 Directory.CreateDirectory(testFolder);
                 DummyBuildEngine dummyEngine = new DummyBuildEngine();
-                TaskExecutionContext taskContext = new TaskExecutionContext(testFolder, 0 /* returns failure code */ );
+                TaskExecutionContext taskContext = new TaskExecutionContext(testFolder, testFolder, testFolder, 0 /* returns failure code */ );
```

Comment:
There are now three paths used the task:
* the binary path it should look for the build wrapper exes in
* the config path where it should look for the config file, and
* the build wrapper output path that should be specified in the config file.
Not all tests require all of these values, depending on how early the test is expected to fail.